src/includes/load-service.h

```diff
@@ -467,6 +467,23 @@ inline gid_t parse_gid_param(unsigned line_num, const std::string &param, const
     return grent->gr_gid;
 }
 
+// Parse a permission param, specified as a decimal number of permission (such as 0600)
+inline int parse_perms(unsigned line_num, string &paramval, const std::string &servicename,
+        const char * paramname)
+{
+    std::size_t ind = 0;
+    try {
+        int perms = std::stoi(paramval, &ind, 8);
+        if (ind != paramval.length()) {
+            throw std::logic_error("");
+        }
+        return perms;
+    }
+    catch (std::logic_error &exc) {
+        throw service_description_exc(servicename, std::string(paramname) + ": badly-formed or "
+                "out-of-range numeric value", line_num);
+    }
+}
```

Comment:
There should be a blank line between this method and the next one (as per style guide).